### PR TITLE
Allows All Species to Select Certain Neutral Traits

### PR DESCRIPTION
**Pull request by KasparoVy**
You can now pick the autohiss, bloodsucker, succubus, hard vore, trashcan, expensive taste and glowing eyes neutral traits as any species.

virgo ss13 neutral traits for all 20200606

### CODE REVIEW

**Comment by KasparoVy** [comment]
RIP, forgot to set the PR name at first.

File: vorestation.dme [hunk 1]
```diff
@@ -1643,9 +1643,9 @@
 #include "code\modules\ai\ai_holder_targeting_vr.dm"
 #include "code\modules\ai\interfaces.dm"
 #include "code\modules\ai\say_list.dm"
-#include "code\modules\ai\aI_holder_subtypes\simple_mob_ai.dm"
-#include "code\modules\ai\aI_holder_subtypes\simple_mob_ai_vr.dm"
-#include "code\modules\ai\aI_holder_subtypes\slime_xenobio_ai.dm"
+#include "code\modules\ai\ai_holder_subtypes\simple_mob_ai.dm"
```
Comment (KasparoVy) [inline vorestation.dme:1646]: Man this keeps getting reverted to `aI` lmao.

File: code/modules/mob/living/carbon/human/species/station/blank_vr.dm [hunk 2]
```diff
@@ -8,6 +8,7 @@
 	var/eat_minerals = FALSE //HEAVY METAL DIET
 	var/base_species = null // Unused outside of a few species
 	var/selects_bodytype = FALSE // Allows the species to choose from body types intead of being forced to be just one.
+	var/list/traits = list() //Contains select neutral traits only unless you're a custom species.
```
Comment (Arokha) [inline code/modules/mob/living/carbon/human/species/station/blank_vr.dm:11]: Uhhhh species other than custom species are global singletons. Every 'Tajaran' for example is a reference to a single global instance of `/datum/species/tajaran`. You can't just add traits to them, *everyone* will have those traits that is that species in the round. Eventually every Tajaran (or whatever) will have every trait suddenly added to them and This Is Bad.

Comment (KasparoVy) [reply code/modules/mob/living/carbon/human/species/station/blank_vr.dm:11]: 😮

Comment (Arokha) [reply code/modules/mob/living/carbon/human/species/station/blank_vr.dm:11]: For further reference see human.dm:1152, in set_species(), where it sets `species` to the singleton in the `GLOB.all_species` list

**Comment by KasparoVy** [comment]
Nope. Didn't consider that we didn't set species with `new`'d types, lots of prerequisites. TBD after that's sorted.

Thanks for flagging it @Arokha

File: code/modules/mob/living/carbon/human/species/station/blank_vr.dm (hunk 2 above)
Comment (KasparoVy) [reply code/modules/mob/living/carbon/human/species/station/blank_vr.dm:11]: Spot on! I looked that and git grepped for everywhere else species is set

I'll need to go over it with a fine toothed comb as the last time I saw species anti-globalized it was really involved

**Comment by Novacat** [comment]
https://github.com/VOREStation/VOREStation/pull/4839

I would recommend looking at this PR perhaps on how to do this?

**Comment by KasparoVy** [comment]
That's an interesting approach @Novacat -- thanks for letting me know!

**Pull request closed without merging**